examples/atari/a3c/README.md

```diff
@@ -0,0 +1,116 @@
+# A3C
+This example trains an Asynchronous Advantage Actor Critic (A3C) agent, from the following paper: [Asynchronous Methods for Deep Reinforcement Learning](https://arxiv.org/abs/1602.01783). 
+
+## Requirements
+
+- atari_py>=0.1.1
+- opencv-python
+
+## Running the Example
+
+```
+python train_a3c.py [options]
+```
+
+### Useful Options
+- `--gpu`. Specifies the GPU. If you do not have a GPU on your machine, run the example with the option `--gpu -1`. E.g. `python train_dqn.py --gpu -1`.
+- `--env`. Specifies the environment. 
+- `--render`. Add this option to render the states in a GUI window.
+- `--seed`. This option specifies the random seed used.
+- `--outdir` This option specifies the output directory to which the results are written.
+
+To view the full list of options, either view the code or run the example with the `--help` option.
+
+## Results
+These results reflect ChainerRL  `v0.6.0`. The ChainerRL score currently consists of a single run. The reported results are compared against the scores from the [Noisy Networks Paper](https://arxiv.org/pdf/1706.10295.pdf), since the original paper does not report scores for the no-op evaluation protocol.
+
+We use the best intermediate scores on each domain to evaluate A3C.
+
+We aim to follow the original paper's evaluation protocol as closely as possible.
+
+| Results Summary ||
+| ------------- |:-------------:|
+| Number of seeds | 1 |
+| Number of common domains | 52 |
+| Number of domains where paper scores higher | 25 |
+| Number of domains where ChainerRL scores higher | 24 |
+| Number of ties between paper and ChainerRL | 3 | 
+
+
+
+| Game        | ChainerRL Time           |
+| ------------- |:-------------:|:-------------:|
```

Comment:
Source: Noisy Networks paper: https://arxiv.org/pdf/1706.10295.pdf , page 18

Reply:
Please double check. I can always make a mistake.

Reply:
Can you add source to README as well?